docs/documentation/server_admin/topics/identity-broker/post-login-flow.adoc

```diff
@@ -0,0 +1,44 @@
+[[_identity_broker_post_login_flow]]
+
+=== Post login flow
+
+Post login flow is useful for the situations, when you want to trigger some additional authentication actions after every login with the particular identity provider.
+For example you may want to trigger 2-factor authentication after every login of {project_name} to `Facebook` because `Facebook` does not provide 2-factor authentication during their login.
+
+Once you setup the authentication flow with the needed steps, it is needed to set it as `Post login flow` when configuring the identity provider.
+
+==== Post login flow examples
+
+===== Requesting 2-factor authentication after identity provider login
+
+The easiest way is to enforce authentication with one particular 2-factor method. For example when requesting OTP, the flow can look like this with only single authenticator configured.
+Such flow will ask user to configure his OTP during first login with the identity provider for the case when user does not have OTP set on his account.
+
+.2FA post login flow with OTP
+image:images/post-login-flow-otp.png[Post login OTP]
+
+The more complex setup can include multiple 2-factor authentication methods configured as `ALTERNATIVE`. In this case it may be needed to make sure that user is requested to setup one of
+the methods if he does not yet have any 2-factor authentication configured on his account. This could be done by:
```

Comment:
```suggestion
The more complex setup can include multiple 2-factor authentication methods configured as `ALTERNATIVE`. In this case, make sure that the user is requested to setup one of
the methods if that user does not yet have any 2-factor authentication configured on the account. This could be done by:
```